integration/utils/contLoadGenerator.go

```diff
@@ -51,7 +51,6 @@ type ContLoadGenerator struct {
 	availableAccounts    chan *flowAccount                             // queue with accounts available for   workers
 	happeningAccounts    chan func() (*flowAccount, string, time.Time) // queue with accounts happening after worker processing
 	txTracker            *TxTracker
-	txStatsTracker       *TxStatsTracker
```

Comment:
have we tested that the loader runs without this field?

Reply:
yep.  test, build, and manual `make load`